### ISSUE TITLE
Unexpected Behavior Upgrading Plugins

### ISSUE THREAD

**Issue by geirman** [cfwheels/cfwheels#199]
I'm not sure if this occurs with all plugins or just dbmigrate, but the following test cases are with dbmigrate only. 
1. With 1.1.8, add the dbmigrate 1.0.1 plugin to the plugins directory then reload framework
   RESULT (FAIL): dbmigrate v0.9 (not upgraded)
2. Upgrade wheels to 1.2 RC1, reload framework
   RESULT (SUCCESS): dbmigrate v1.0.1 (upgraded)
3. Without doing anything to the plugin directory, roll back wheels to 1.1.8 and reload framework
   RESULT (FAIL): dbmigrate v0.9 (downgraded?!)

NOTE: In all of the above steps, both dbmigrate-0.9.zip and dbmigrate-1.0.1.zip existed in the plugins directory.
1. With wheels 1.1.8 installed, delete dbmigrate-0.9zip from plugins directory and reload the framework
   RESULT (SUCCESS): dbmigrate 1.0.1 (upgraded)

In summary, it seems that dbmigrate will not upgrade unless wheels is also upgraded unless the upgraded plugin is it's only option.
